packages/client/src/util/toDataValue.ts

```diff
@@ -55,10 +55,6 @@ export async function toDataValue(
       {
         filename: value.name,
       },
-      {
-        "Content-Length": value.size.toString(),
-        "Content-Type": value.type,
```

Comment:
So obviously in this case, we just want to use whatever the blob's content-type is. After all, we're asking it.

However, the foundry platform sdk is used by customers too and they may at some point end up with a blob that has a content type they don't want. Lets say they get an upload for a json file but they need the content type in the system to be something specific? 

Therefore, I would like to raise the question: Should we make `Content-Type` optional and fill it with the blob value if not provided?

Reply:
I see the use case, and I probably agree with the solution. The alternative would be to just enforce the users attach a Blob with the correct type attached, but that's probably more overhead than we should pass to the consumers and also requires them to make a copy. I don't think there's a scenario in which the Blob's actual attached content type is transmitted through the wire or used in the generated SDK's, so this should be fine